lib/avalon/batch/manifest.rb

```diff
@@ -25,14 +25,51 @@ class Manifest
       SKIP_FIELDS = [:collection]
 
       def_delegators :@entries, :each
-      attr_reader :spreadsheet, :file, :name, :email, :entries, :package
+      attr_reader :spreadsheet, :file, :name, :email, :entries, :package, :manifest_logger
 
       class << self
         def locate(root)
+          locate_with_status(root)[:processable]
+        end
+
+        def locate_with_status(root)
+          status = {
```

Comment:
Yes, you can be in multiple states at the same time, since the same collection can have multiple manifest files in the same directory (and some might be processed, some might be errored, some might be xlsx, some might be csv, etc.). This is the upstream design.